### PR TITLE
Documentation for non-public cloud support for application insight sc…

### PR DESCRIPTION
**Pull request by v-shenoy**
…aler.

Signed-off-by: Vighnesh Shenoy <vshenoy@microsoft.com>

Update application insights scaler documentation.
https://github.com/kedacore/keda/pull/2735

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.7/scalers/azure-app-insights.md [hunk 1]
```diff
@@ -40,6 +46,9 @@ for further details.
 - `activeDirectoryClientId` - Id of the Active Directory client. The client must have `Monitoring Reader` permissions for the Application Insights instance.
 - `activeDirectoryClientPassword` - Password of the Active Directory client password.
 - `metricFilter` - Further specify the metrics query using a filter. For example `cloud/roleName eq 'example`. (Optional)
+- `cloud` - Name of the cloud environment that the Event Hub belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`)
```
Comment (JorTurFer) [inline content/docs/2.7/scalers/azure-app-insights.md:49]: Please, add `Optional` if the field is optional to follow the pattern
- name - Description. (Values: x, y, z, Default: y, Optional, Extra Info)

**Review by JorTurFer — COMMENTED** [review]
LGTM!
Only one comment inline!

File: content/docs/2.7/scalers/azure-app-insights.md (hunk 1 above)
Comment (v-shenoy) [reply content/docs/2.7/scalers/azure-app-insights.md]: Also added it for the EventHub scaler as I missed it there.

File: content/docs/2.7/scalers/azure-app-insights.md [hunk 2]
```diff
@@ -40,6 +46,9 @@ for further details.
...
+- `appInsightsResourceURL` - Application Insights REST API url of the cloud environment. (Required when `cloud` is set to `private`, e.g. `https://login.chinacloudapi.cn/` for `AzureChinaCloud`).
```
Comment (JatinSanghvi) [inline content/docs/2.7/scalers/azure-app-insights.md:50]: Change case so that the words are 'URL' and 'Private'.

**Review by JorTurFer — APPROVED** [review]
LGTM!
Thanks ❤️

**Review by zroubalik — APPROVED** [review]

**Pull request merged**